frontend/cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+describe("List CANs", () => {
+    it("list cans", () => {
+        cy.visit("/");
+        cy.contains("/cans").click();
+        cy.url().should("include", "/cans");
+    });
+});
+
+describe("CAN Details", () => {
+    it("get can details", () => {
+        cy.visit("/cans");
+        cy.contains("G99PHS9").click();
+        cy.url().should("include", "/cans/3");
+    });
+});
+
+describe("Select Fiscal Year", () => {
+    it("get can fiscal year details", () => {
+        cy.visit("/cans/3");
+        cy.contains("G99PHS9").click();
+        cy.wait(100);
```

Comment:
You shouldn't need to call `.wait` since Cypress automatically waits for the DOM to elements to fill in.

Reply:
I don't disagree, but was actually noticing, on a first page load, if you go directly. The page loads without the `canDetails` and then quickly refreshes with the `canDetails` populated. So on first render, the DOM isn't the correct on, it needs to wait for the `/cans/3` to be called and update the components. This was my observation at least.

Reply:
Tried without, and works this time, might have been how I was grabbing the dom object my first time 🤷

Reply:
Yeah, that's OK to Cypress.  :D  Cypress checks repeatedly for a (configurable) amount of time to see if what it is looking for shows up.  So, even if it doesn't show up on first render, Cypress will wait patiently until the element/text/whatever does show up.